packages/monaco/src/browser/monaco-outline-contribution.ts

```diff
@@ -64,6 +68,38 @@ export class MonacoOutlineContribution implements FrontendApplicationContributio
                 });
             }
         });
+
+        this.editorManager.onCurrentEditorChanged(widget => {
+            if (widget) {
+                this.addConfigTabbingWidget();
+
+                widget.editor.onEditorOptionsChanged(() => {
+                    this.removeConfigTabbingWidget();
+                    this.addConfigTabbingWidget();
+                });
+            } else {
+                this.removeConfigTabbingWidget();
+            }
+        });
+    }
+
+    protected addConfigTabbingWidget() {
+        const editor = this.editorManager.currentEditor;
+        if (editor && this.getModel(editor)) {
+            const modelOptions = this.getModel(editor).getOptions();
+            const tabSize = modelOptions.tabSize;
+            const useSpaceOrTab = modelOptions.insertSpaces ? 'Spaces' : 'Tabs';
+            this.statusBar.setElement('editor-status-tabbing-config', {
+                text: `${useSpaceOrTab}: ${tabSize}`,
+                alignment: StatusBarAlignment.RIGHT,
+                priority: 10,
+                command: EditorCommands.CONFIG_TABBING.id,
+                saveLayout: false
```

Comment:
Just for informations, this was added because otherwise the widget would get reloaded and even though the editors would get reloaded also, unless you explicitly click on an editor, active or current will be undefined, thus clicking on this status bar right as the app reloaded would cause the command to fail. That was it right @elaihau ?

Reply:
Yes you are right. 
Setting saveLayout to false means that particular statusBarElement won't be saved or show up the next time theia gets reloaded.

Reply:
please remove `saveLayout` as it was before, first try to solve an issue with existing abstractions before changing them

You can try to remove `editor-status-tabbing-config` element on `FrontendApplicationContribution.onStart` callback.

Reply:
Thank you for the feedback @akosyakov  :)

I find that the `FrontendApplicationContribution.onStart` is called before`statusBar.setLayoutData()`, therefore I am unable to remove that status bar entry directly from `StatusBarImpl.entries`. 

I also tried to call `statusBar.removeElement()` from the `EditorContribution.onStart()`. And it did not work either. Because the statusBar had no entries at the point that `onStart()` was called. 

Besides, I tried to hook my code with the `onDidCreateModel` and `onDidCreateEditor` events. Neither worked, because they both were fired before `statusBar` got any entries.

Therefore, I added `onStatusBarEntriesLoaded: Event` to `StatusBar`, making it fire once all entries are loaded. While this solution leaves `interface StatusBarEntry` untouched, it changes another interface `StatusBar`. I am sort of running of ideas for now :) Could you please advise what's the best way to do it? Thank you !